### DISCUSSION TITLE
Where is the dataset file saved ?

### DISCUSSION THREAD

**Discussion by Andr-ea** [Jisco/VisualFamilyTree#27] [Q&A] [answered]
Hi, I would like to ask if I can know where the dataset file that is saved when I exit is saved.
I don't mean the file generated by the export but the dataset that is loaded at each startup.

Thank you very much

**Comment by Jisco** [comment, marked as answer]
Hi sure,
the files are here:: "C:\Users\{USERNAME}\AppData\Roaming\Visual Family Tree\databases".
All databases with identifiers are stored in the file “families.json” and the corresponding data is located under {identifier}.json.

May I ask what you need this for?!

**Reply by Andr-ea** [reply]
Thank you very much, it was just to know how the nodes and connections were saved.
